### ISSUE TITLE
0.37 rc1 source_files not finding files with spaces in

### ISSUE THREAD

**Issue by mxcl** [CocoaPods/CocoaPods#3471]
I cannot lint with 0.37, but 0.36.4 does lint.

```sh
 -> PromiseKit (1.5.3)
    - ERROR | [PromiseKit/Swift/Promise, PromiseKit/Swift/CloudKit, PromiseKit/Swift/UIKit, and more...] The `source_files` pattern did not match any file.

[!] The podspec does not validate.
```

The problem with these subspecs is they reside in a directory with spaces in. In case that is the problem. Here's the podspec: https://github.com/mxcl/PromiseKit/blob/master/PromiseKit.podspec

**Comment by neonichu** [comment]
hm, can't repro this:

```bash
$ xcodebuild -version
Xcode 6.3.1
Build version 6D1002
$ pod --version
0.37.0.rc.1
$ pod lib lint PromiseKit.podspec 

 -> PromiseKit (1.5.3)

PromiseKit passed validation.

[!] The platform of the target `Pods` (OS X 10.9) may not be compatible with `PromiseKit/Swift/CloudKit (1.5.3)` which has a minimum requirement of iOS 8.0 - OS X 10.10.
```

**Comment by mxcl** [comment]
I am also using Xcode 3.6.1.

Notably, in case it makes a difference, I was running `pod trunk push PromiseKit.podspec`

**Comment by segiddins** [comment]
I see the issue here, and it has nothing to do with the spaces, but rather me doing a bad job trying to clean the cached directory. I'm trying to pull together a fix right now

**Comment by segiddins** [comment]
Long story short: the issue was having a subspec with a higher minimum deployment target than the root spec.

**Comment by mxcl** [comment]
Interesting, anything I *should* do? I've played with various arrangements of deployment targets in the past in an attempt to please the linter. I'd happily not have the root spec have a deployment target and instead have it determined by the subspecs, but I think that doesn't lint.

**Comment by segiddins** [comment]
@mxcl no no no you did nothing wrong :) It's entirely a CocoaPods bug, one that I have a fix nearly done for

**Comment by kylef** [comment]
@mxcl This has been fixed in the new 0.37.0.rc.2 release, please upgrade :).

**Comment by mxcl** [comment]
Thanks everyone!
